tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -147,6 +147,34 @@ func createTempFile() string {
 	return fileName
 }
 
+// executing test
+func executeTest(flags []string, m *testing.M) (successCode int, err error) {
+	for i := 0; i < len(flags); i++ {
+		if err = mountGcsfuse(flags[i]); err != nil {
+			log.Printf("mountGcsfuse: %v\n", err)
+			return
+		}
+
+		log.Printf("Test log: %s\n", logFile)
+
+		// Creating a temporary directory to store files
+		// to be used for testing.
+		tmpDir, err = os.MkdirTemp(mntDir, "tmpDir")
```

Comment:
do you need a seperate tmpDir for every mounnt?

Reply:
Yes because for every mounting after running test we are removing all the files and dirs using os.RemoveAll.

Reply:
should I remove os.RemoveAll and create tempdir only when i=0(first mount) ?

Reply:
yes, instead of doing i=0 check create the tempDir in TestMain funnctionn.